### DISCUSSION TITLE
Desktop icons always resetted after Explorer restart

### DISCUSSION THREAD

**Discussion by senyaak** [valinet/ExplorerPatcher#2495] [Q&A]
Hi all, I have issue with my desktop icons. Every time the explorer starts the icons are messed up. I tried a lot of things I found on microsoft forums(deleted icons cache, ran healthrestore etc ), but now I think it could be issue of exporerPatcher. 
I have 2 monitors and have 150% scaling on.
Anyone had the same issue?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please show a screen shot of what you are reporting from before and then after it has happened.

**Reply by senyaak** [reply]
WV: 22h2, OS 22621.2715
EPV: 22621.2506.60.1
Settings: [ExplorerPatcher_22621.2506.60.1.txt](https://github.com/valinet/ExplorerPatcher/files/13450463/ExplorerPatcher_22621.2506.60.1.txt)
Before: 
![image](https://github.com/valinet/ExplorerPatcher/assets/15360093/afae4e07-2355-4613-8aa1-5e59ad3515dd)
After: 
![image](https://github.com/valinet/ExplorerPatcher/assets/15360093/cefca180-d62d-4623-b6d6-3059430b3056)

All the icons moved to the secondary screen

**Comment by pyrates999** [comment]
Do you have register as a shell extension enabled in EP? If so, disable it, and reboot.

**Comment by senyaak** [comment]
> Do you have register as a shell extension enabled in EP? If so, disable it, and reboot.

Nope - its disabled

**Comment by pyrates999** [comment]
Ok, then uninstall EP, reboot, and do the following:

1. open up task manager
2. go to details tab
3. right click on all instances of explorer.exe and choose end task
4. go to file, run, and put in explorer.exe

See if this is still happening.

**Comment by Amrsatrio** [comment]
Uninstall EP and see if this problem persists. I don't daily drive a multi monitor setup but, if one of your monitors use DisplayPort I guess that is the culprit, especially LG monitors. They have a DP deep sleep function which "disconnects" your monitor when the display goes into sleep mode/energy saving mode, therefore your desktop icons will mess up. Unfortunately I haven't found any fixes for this.
